codeforamerica/vita-min
# re-encrypts heroku credentials with new key

## Link to pivotal/JIRA issue
n/a

## Is PM acceptance required? (delete one)
No - merge after code review approval

## What was done?
- Re-encrypted the heroku env creds w/ a new encryption key
- Added the updated key to lastpass
- Updates the build script for Heroku environments to reference the new key with a more heroku-specific naming convention.
- After merge, will need to purge the old key from github secrets.

## How to test?
- Pull the branch
- Copy the new encryption key from lastpass into `config/credentials/heroku.key`
- Ensure you can decrypt the file
